facebook-commerce.php

```diff
@@ -4256,6 +4261,27 @@ function delete_product_item( $wp_id ) {
 		}
 	}
 
+
+	/**
+	 * Uses the Graph API to delete the Product Group associated with the given product.
+	 *
+	 * @since 2.0.0-dev.1
+	 *
+	 * @param int $product_id product ID
+	 */
+	private function delete_product_group( $product_id ) {
+
+		$product_group_id = $this->get_product_fbid( self::FB_PRODUCT_GROUP_ID, $product_id );
+
+		if ( $product_group_id ) {
+
+			$pg_result = $this->fbgraph->delete_product_group( $product_group_id );
```

Comment:
Should we not be using our API to delete the product group?

Reply:
I think can do that now, but I never added stories for replacing those calls due to the limited. time we have and the things we still have left to do, so I've been replacing only what is currently causing other issues.

I added a TODO note in af3b7c1. There are other usages of the old API methods, so is probably better to clean those up at the same time later.

What do you think?